src/v4/injectGlobalToCreateGlobalStyle.js

```diff
@@ -1,17 +1,18 @@
 "use strict";
 
 const t = require("babel-core").types;
-const { prop } = require("lodash/fp");
+const { prop, set } = require("lodash/fp");
 const { manipulateOptions } = require("./common");
 
 const WARNING =
   "An injectGlobal usage was converted to createGlobalStyles via codemod but needs to be hooked up. See styled-components.com/docs/api#createglobalstyle for instructions.";
-
 const PREVIOUS = "injectGlobalStyle";
```

Comment:
It's just "injectGlobal", not "injectGlobalStyle"

Reply:
Haha yep, thanks.